lib/main.js

```diff
@@ -43,6 +43,11 @@ app.use('/micropub/:targetsite', micropub({
   tokenReference: function (req) {
     return (req.targetsite.token || []).concat(config.token);
   },
+  queryHandler: function (q, req) {
+    if (q === 'syndicate-to' && req.targetsite.syndicateTo) {
+      return { 'syndicate-to': req.targetsite.syndicateTo };
```

Comment:
If added a check for whether `config.handlerOptions.syndicateTo` is set, added that option to `lib/config.js` mimicking how `deriveCategory` works + changed it so that the value of that config variable is returned rather than the hard coded value, then we would be all set 👍

Do you want to do that or should I?